docs/pages/push-notifications/receiving-notifications.mdx

```diff
@@ -34,6 +36,180 @@ useEffect(() => {
 }, []);
 ```
 
+<Collapsible summary="Android Notification Object Example from addNotificationReceivedListener">
+
+Here's a sample of the notification object received by the callback function when using `Notifications.addNotificationReceivedListener`:
```

Comment:
```suggestion
Sample of the `notification` object received by the callback function when using `Notifications.addNotificationReceivedListener`:
```

We prefer using contractions with words like "you" or "we". For rest, we should avoid them. Also, the sample snippet below is using a mix of different code languages with JSX marked as primary code language (which is not accurate in this case. Let's remove the console log statement and instead use code syntax highlight and use comments in the JSON snippets.